stable/confidant/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: A Helm chart for Confidant (github.com/lyft/confidant)
 name: confidant
 icon: https://github.com/lyft/confidant/raw/gh-pages/images/safe-1d911346.png
-version: 1.3.0
+version: 1.3.1
```

Comment:
Current `master` is at `1.3.1` [already](https://github.com/fairfaxmedia/charts/blob/master/stable/confidant/Chart.yaml#L5) - you need to go to a later number. Given you are adding a new feature, I'd suggest `1.4.0`